src/js/service/PiskelApiService.js

```diff
@@ -151,6 +154,8 @@
    */
   ns.PiskelApiService.prototype.loadSpritesheet = function (uri, frameSizeX,
       frameSizeY, frameRate) {
+    $.publish(Events.LOAD_NEW_PISKEL);
```

Comment:
Instead of publishing this here, at the `PiskelApiService` layer, would it be appropriate to do this in `ns.PiskelController.prototype.setPiskel`?  `createNewPiskel` calls it directly and `loadSpritesheet` calls it a few layers down - it seems to be a 'chokepoint' for loading a new image/project into the editor, and there are lots of other places that call it.

Oh, I found a downside: `ResizeController` uses `setPiskel` on resize (it literally creates a new Piskel object at the new size and swaps it in), and it looks like `HistoryService` actually calls `setPiskel` when restoring a previous state - so doing this in `setPiskel` would cause you to lose your history whenever you did either of those things.

So, I guess this is good for now?  I'd merge this for our purposes, then open it upstream and see if we can get a conversation going about it there.  I get the impression that, internal to Piskel, undoing across a load event is fine and maybe down the road we actually want to support that within GameLab - we would just need to figure out how to go the other way and propagate a 'reverse-load' event to the animation tab, and change the selected animation.  I'd rather do that later though.